Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs

```diff
@@ -942,6 +967,17 @@ private void GetComponentByTypeAndName(Type type, string managerName, out IMixed
                     manager = MixedRealityComponents[i].Item2;
                     break;
                 }
+
+                var interfaces = MixedRealityComponents[i].Item2.GetType().GetInterfaces();
```

Comment:
Seems like the same lines over and over again. Could we make this into it's own thing?

Reply:
Probably